docs/Yaml/testSettings.md

```diff
@@ -6,6 +6,7 @@ This is used to define settings for the tests in the test plan
 
 | Property | Required | Description |
 | -- | -- | -- |
+| locale | Yes | The locale/culture syntax in which the test cases or test steps are written in. See {TODO:shortlink} for more info. If unspecified, `CultureInfo.CurrentCulture` will be used for the locale by default for parsing the test steps. |
```

Comment:
Note there's a TODO here.

Reply:
updated this with a doc URL pointing to Global Support in Power Fx